### PR TITLE
Send an email to addon owners after their addon got auto signed (bug 1070227)

### PR DESCRIPTION
**Pull request by magopian**
Fixes [bug 1070227](https://bugzilla.mozilla.org/show_bug.cgi?id=1070227)

Here's the content of the mail at the moment:

```
Subject: Mozilla Add-ons: tabsmacka has been automatically signed on AMO
From: Mozilla Add-ons <nobody@mozilla.org>
To: foo@example.com
Reply-To: amo-editors@mozilla.org


Your add-on, tabsmacka, has been automatically signed for distribution in
upcoming versions of Firefox. The signing process involved repackaging the
add-on files and adding the string '.1-signed' to their versions numbers. The
new versions have kept their review status and are now available for your
users.
We recommend that you give them a try to make sure they don't have any
unexpected problems: http://localhost:8000/developers/addon/tabsmacka/versions.

If you are unfamiliar with the extension signing requirement, please read the
following documents:

* Signing announcement:
  http://blog.mozilla.org/addons/2015/02/10/extension-signing-safer-experience/

* Documentation page and FAQ: https://wiki.mozilla.org/Addons/Extension_Signing

If you have any questions or comments on this, please reply to this email or
join #amo-editors on irc.mozilla.org.

You're receiving this email because you have an add-on hosted on
https://addons.mozilla.org.
```

### CODE REVIEW

**Comment by magopian** [comment]
@jvillalobos what do you think of the wording?

**Comment by jvillalobos** [comment]
I think the second paragraph "Only versions..." can be left out. Might confuse some people.

File: lib/crypto/tasks.py [hunk 1]
```diff
@@ -21,6 +22,28 @@
 # Minimum Firefox version for not default to compatible addons.
 MIN_NOT_D2C_VERSION = '37'
 
+EMAIL_SUBJECT = 'Mozilla Add-ons: {addon} has been automatically signed on AMO'
```
Comment (clouserw) [inline lib/crypto/tasks.py]: Do these strings need to be u""?  (for an add-on with a unicode name)

**Comment by clouserw** [comment]
patch looks good to me w/ IRC comments

**Comment by kmaglione** [comment]
"have kept their review status"

"We recommend that you give them a try to make sure that they don't have any unexpected problems:"

The indentation at the end is a bit odd. Also, there needs to be an acknowledgement at the end that explains why they're getting an email (that they have an add-on hosted on AMO, and we're contacting them per notification settings)

**Pull request merged**